pluto_build/sql/corr_communitydistrict.sql

```diff
@@ -14,6 +14,20 @@ WHERE a.bbl = b.bbl
 	AND b.field = 'cd'
 	AND a.bbl NOT IN (SELECT bbl FROM pluto_corrections WHERE field = 'cd');
 
+INSERT INTO pluto_corrections_not_applied
```

Comment:
It won't work because the values will no longer match (I suppose maybe it could be switched to new_value <> a.cd but that won't catch things where the value was updated in the underlying datasource